### PR TITLE
Sequencer repo status

### PR DESCRIPTION
**Pull request by bpeabody**
Addresses card: https://github.com/twosigma/git-meta/projects/2#card-8136879

Dependent on https://github.com/twosigma/git-meta/pull/515

### CODE REVIEW

File: node/lib/util/print_status_util.js [hunk 1]
```diff
@@ -397,6 +398,39 @@ A cherry-pick is in progress.
...
+ * @param {SequencerState.TYPE} type
+ * @return {String}
+ */
+exports.getSequencerCommand = function (type) {
+    const TYPE = SequencerState.TYPE;
+    switch (type) {
+    case TYPE.CHERRY_PICK: return "cherry-pick";
+    case TYPE.MERGE: return "merge";
+    case TYPE.REBASE: return "rebase";
+    default: assert(false, "shouldn't get here"); break;
```
Comment (novalis) [inline node/lib/util/print_status_util.js]: "unhandled sequencer type " + type + " in switch"

File: node/lib/util/status_util.js [hunk 2]
```diff
@@ -663,4 +670,12 @@ Before proceeding, you must complete the cherry-pick in progress (by running
 Please resolve outstanding conflicts before proceeding:
 ${PrintStatusUtil.printRepoStatus(status, "")}`);
     }
+    if (null !== status.sequencerState) {
+        const command =
+               PrintStatusUtil.getSequencerCommand(status.sequencerState.type);
+        throw new UserError(`\
+Before proceeding, you must complete the ${command} in progress (by running
```
Comment (novalis) [inline node/lib/util/status_util.js]: This is a lot of repeated text from above.  Is there any way to unify this text with the above text?

Comment (bpeabody) [reply node/lib/util/status_util.js]: When this refactoring is over that will all be gone.

Comment (bpeabody) [reply node/lib/util/status_util.js]: As merge, rebase, and cherry-pick will be redone in terms of the sequencer. The next few PRs after this one will contain a lot of deleted code!

**Pull request merged**